submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,345 @@
+/*Правило для тіла сторінки*/
+html, body {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+
+/*Правило для шапки*/
+.header {
+    display: flex;
+    justify-content: flex-end;
+    padding: 20px 0;
+    position: relative;
+}
+/*---------------------------------------------------------------------------
+Правило для меню*/
+.top_menu {
+    display: flex;
+    padding: 0 15px;
+    align-items: center;
+}
+/*Текстові блоки меню*/
+.top_menu__item {
+    line-height: 30px;
+    padding: 0 10px 0 10px;
+    font-size: 14px;
+    text-decoration: none;
+    color: rgba(0, 0, 0, 0.75);
+}
+/*Поведінка меню при наведенні курсора*/
+.top_menu__item-container:hover {
+    text-decoration: underline;
+    /*opacity: 85%;*/
+}
+/*Елемент меню - повідомлення*/
+.top_menu__item_bell {
+    margin-left: 5px;
+    width: 20px;
+    height: 25px;
+    display: block;
+    cursor: pointer;
+    background: url("../icons/bell.png") center center no-repeat;
+    background-size: 20px 25px;
+}
+/*Елемент меню - профіль користувача*/
+.top_menu__item_user {
+    display: block;
+    height: 32px;
+    width: 32px;
+    border-radius: 50%;
+    background: url("../icons/favicon.png") center center no-repeat;
+    background-size: 32px 32px;
+}
+/*---------------------------------------------------------------------------
+Чекбокс попапа*/
+.input_popup {
+    position: absolute;
+    opacity: 0;
+}
+/*Видимість попапа при натисканні на чекбокс*/
+.input_popup:checked ~ .popup {
+    visibility: visible;
+}
+/*Попап - основна частина*/
+.popup {
+    visibility: hidden;
+    display: flex;
+    width: 320px;
+    height: 440px;
+    position: absolute;
+    box-sizing: border-box;
+    top: 62px;
+    right: 15px;
+    border: 1px solid #ccc;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+    background: #ffffff;
+    flex-direction: column;
+    z-index: 50;
+    margin: 0;
+}
+
+/*Псевдокласи для стрілки попапа*/
+.popup:after,
+.popup:before {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-color: transparent;
+  border-style: dashed dashed solid;
+  border-width: 0 8.5px 8.5px;
+  right: 102.5px;
+  border-bottom-color: rgba(0, 0, 0, 0.2);
+  top: -10px;
+}
+.popup:after {
+  border-color: transparent;
+  border-bottom: solid #fff;
+  border-style: dashed dashed solid;
+  border-width: 0 8.5px 8.5px;
```

Comment:
8.5 px? wow, why do you need such accuracy?

Reply:
Just tried to align it in the center of the icon %)

Reply:
Done